src/SpamKiller.ts

```diff
@@ -267,12 +342,15 @@ export default class SpamKiller {
             console.log(`SpamKiller: ${message.author.username}#${message.author.discriminator}'s message triggered our spam detector, but they've got ${member.roles.cache.size} roles. (https://discordapp.com/channels/${message.guild?.id}/${message.channel.id}/${message.id})`);
             return;
         }
+        if (message.channel instanceof Discord.ThreadChannel && (message.channel.parentId == "978519681184964629" || message.channel.parentId == "978514449352777798" || message.channel.parent instanceof Discord.ForumChannel)) {
+            return console.log(`SpamKiller: ${message.author.username}#${message.author.discriminator}'s message triggered our spam detector, but channel has an exemption. (https://discordapp.com/channels/${message.guild?.id}/${message.channel.id}/${message.id})`);
+        }
```

Comment:
This was to stop botty from deleting the original post in looking-for-work and looking-for-devs posts